simtools/util/names.py

```diff
@@ -949,7 +949,9 @@ def corsikaSubLogFileName(arrayName, site, primary, run, mode, label=None):
     """
     name = 'log-sub-corsika-run{}-{}-{}-{}'.format(run, arrayName, site, primary)
     name += '_{}'.format(label) if label is not None else ''
-    name += '-' + mode + '.log'
+    name += '.log'
+    if len(mode)>0:
+        name += "." + mode
```

Comment:
Nowhere in the code did we use `.` to separate info in file names, we always used `-` or `_`. It's probably best not to mix a new convention, no?

Reply:
OK - although we are writing so many files to disk with quite a lot of differences in style (e.g. `-` vs `_`).